packages/sidenav/src/SidenavItem.ts

```diff
@@ -166,6 +166,18 @@ export class SideNavItem extends LikeAnchor(Focusable) {
             parentSideNav.startTrackingSelectionForItem(this);
             this.selected =
                 this.value != null && this.value === parentSideNav.value;
+            if (
+                this.selected === true &&
+                parentSideNav.variant === 'multilevel'
+            ) {
+                let element = this.parentElement;
+                while (element instanceof SideNavItem) {
+                    if (element.expanded === false) {
+                        element.expanded = true;
+                    }
```

Comment:
Let's collapse this.
```suggestion
                    element.expanded = true;
```